scripts/install-nix-from-closure.sh

```diff
@@ -65,6 +65,21 @@ elif [ "x${1:-}" != "x" ]; then
     exit
 fi
 
+if [ "$(uname -s)" = "Darwin" ]; then
+    canondest="/opt/nix"
+    if ! grep -q '^nix' /etc/synthetic.conf 2> /dev/null; then
+        echo "configuring /nix in synthetic.conf" >&2
+        printf "nix\t%s\n" "$canondest" | sudo tee -a /etc/synthetic.conf > /dev/null
+    fi
+    if ! [ -e $dest ] && ! [ -e $canondest ]; then
+        sudo sh -c "mkdir -m 0755 $canondest && chown $USER $canondest"
+    fi
+    # (re)bootstrap the root filesystem.
+    if ! /System/Library/Filesystems/apfs.fs/Contents/Resources/apfs.util -B; then
```

Comment:
Does the bootstrap automatically create the symlink too? It's probably worth double-checking to make sure `$dest` now exists after the bootstrap.